### DISCUSSION TITLE
convert: Incompatible type for "RichTIFFIPTC"; tag ignored. `TIFFFetchNormalTag' @ warning/tiff.c/TIFFWarnings/960.

### DISCUSSION THREAD

**Discussion by guangdao0428** [ImageMagick/ImageMagick#3862] [Help]
1. convert image1.tiff image1.png  
2. warning : convert: Incompatible type for "RichTIFFIPTC"; tag ignored. `TIFFFetchNormalTag' @ warning/tiff.c/TIFFWarnings/960.
3. result :
    no images1.png, but image1-0.png and image1-1.png
   
![image](https://user-images.githubusercontent.com/86390439/123191262-92bd4e80-d4d3-11eb-8d48-bb6bc9c0e805.png)

4.version
  4.1) convert 
        Version: ImageMagick 6.9.10-68 Q16 x86_64 2021-02-24 https://imagemagick.org
        Copyright: © 1999-2019 ImageMagick Studio LLC
        License: https://imagemagick.org/script/license.php
        Features: Cipher DPC Modules OpenMP(3.1)
       Delegates (built-in): bzlib cairo fontconfig freetype gslib jng jp2 jpeg lcms ltdl lzma openexr pangocairo png ps rsvg tiff 
       wmf x xml zlib
   4.2)os
        Linux  3.10.0-957.el7.x86_64 #1 SMP Thu Nov 8 23:39:32 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
5.tiff image
   
[image1.tiff.zip](https://github.com/ImageMagick/ImageMagick/files/6705797/image1.tiff.zip)

6.other
     ImageMagick 7.1.0-1 has same problem

someone  has same question or knows how to resolve ?

thanks for your reply~

**Comment by fmw42** [comment]
That is a warning not an error and should have no effect on the result. The reason you got -0 and -1 outputs is because the TIFF file has more than one page or layer. If you only want the first one, then use 

`convert image1.tiff[0] image1.png`

This is common for tiff files.  

If you do not want to see the warnings, then add -quiet. Note for IM 7, use magick in place of convert

`magick -quiet image1.tiff[0] image1.png`

**Reply by guangdao0428** [reply]
I do it, it works. and what you said help me the reason.

thanks a lot for your help.
